src/RequestExecutor/Response.ts

```diff
@@ -1,23 +1,29 @@
+import { Protocol } from '../Handlers';
+
 export class Response {
+  public readonly protocol: Protocol;
   public readonly status?: number;
   public readonly headers?: Record<string, string | string[]>;
   public readonly body?: string;
   public readonly message?: string;
   public readonly errorCode?: string;
```

Comment:
See https://github.com/NeuraLegion/nexploit-cli/issues/99#issuecomment-769114555

```suggestion
  public readonly protocol: Protocol;
  public readonly status_code?: number;
  public readonly headers?: Record<string, string | string[]>;
  public readonly body?: string;
  public readonly message?: string;
  public readonly error_code?: string;
```